woocommerce/woocommerce-ios
# [Woo POS] Reset cardPresentPaymentInlineMessage when starting a new transaction

Closes: #13275
Touches: #13217


## Description


When we successfully finish a payment and try to do another transaction, for a brief moment we see the "Payment successful" message. This PR fixes the issue by setting `cardPresentPaymentInlineMessage` to `nil`.

## Testing information


- enter POS
- add items to cart
- tap "checkout"
- finish the payment
- tap "New transaction" button to start new order process
- check that the cart is empty
- add items to cart again
- tap "checkout" and make sure that "Payment successful" message is not visible for an instant

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/6242034/603377d4-a884-44c5-829c-d1ad889db055

---
